apps/web/src/services/routes/api.ts

```diff
@@ -176,6 +176,7 @@ export const API_ROUTES = {
                     root: `${documentRoot}/updateDocumentContent`,
                   },
                   evaluations: {
+                    root: `${documentRoot}/evaluations`,
```

Comment:
remove it please, we want flat api REST endpoints that accept an arbitrarely large amount of filters rather than scoping the same resource for each new filter.